lib/workers/branch/execute-post-upgrade-commands.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { mocked } from '../../../test/util';
```

Comment:
```suggestion
import { fs, mocked } from '../../../test/util';
```
Maybe git is there too